WebRoot/js/zimbraMail/abook/view/ZmGroupView.js

```diff
@@ -1646,6 +1656,51 @@ function() {
 	return [(new DwtListHeaderItem({field:ZmItem.F_EMAIL, text:ZmMsg.membersLabel, view:this._view}))];
 };
 
+ZmGroupMembersListView.prototype.getKeyMapName =
+function() {
+	return DwtKeyMap.MAP_MENU;
+};
+
+ZmGroupMembersListView.prototype.handleKeyAction = function(actionCode, ev) {
+
+	if (!this.size()) {
+		return false;
+	}
+
+	var isRowElement = this._focusElement.getAttribute('role') === 'option';
+	var isDeleteElement = this._focusElement.classList.value.includes('ImgDelete');
+
+	if (isDeleteElement) {
+		switch (actionCode) {
+			case DwtKeyMap.CANCEL:
+			case DwtKeyMap.PARENTMENU:
+				var rowElement = this._focusElement.parentElement;
+				while (rowElement.getAttribute('role') !== 'option') {
```

Comment:
if somehow required element is not found then we should add optional breaking condition, to prevent it to go infinite loop

Reply:
Handled the infinite loop behaviour by checking `rowElement` is not null.